cmdutil/service/standard_http_test.go

```diff
@@ -6,8 +6,12 @@ import (
 	"io/ioutil"
 	"net"
 	"net/http"
+	"os"
 	"testing"
 
+	"github.com/jkakar/envtest"
```

Comment:
IMO i don't see much value in importing a package where a couple of functions manipulating the environment would do.

Reply:
Fair enough, I knew we had used envtest in some other projects to deal with tests leaking environment variables, but I see that this repo has a few other cases where they're directly cleaned up w/ the os package. I've updated with f4697806a.